src/main/java/com/icthh/xm/tmf/ms/activation/service/ServiceResourceHelperService.java

```diff
@@ -0,0 +1,10 @@
+package com.icthh.xm.tmf.ms.activation.service;
+
+import com.icthh.xm.tmf.ms.activation.domain.SagaTransaction;
+import com.icthh.xm.tmf.ms.activation.model.v4.Service;
+import com.icthh.xm.tmf.ms.activation.model.v4.ServiceCreate;
+
+public interface ServiceResourceHelperService {
```

Comment:
Can we invent some other meaningful name without _Helper_ and _Service*Service_ like _ResponseEnricher_ or smth?

Reply:
done